node_modules/oae-admin/manageuser/manageuser.html

```diff
@@ -199,6 +209,16 @@ <h3 id="manageuser-modal-title"><!-- --></h3>
     </div>
 --></div>
 
+<div id="manageuser-actions-body-deleteuser-template"><!--
```

Comment:
This would be the desired display for the delete user confirmation dialog:

![screen shot 2015-05-19 at 13 33 55](https://cloud.githubusercontent.com/assets/109850/7713298/b7f40278-fe2b-11e4-8585-57c07c1f6b79.png)